product_pricelist_item_uom/models/product_pricelist_item.py

```diff
@@ -0,0 +1,92 @@
+#  Copyright 2023 Simone Rubino - Aion Tech
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools import float_compare
+
+
+class ProductPricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    uom_id = fields.Many2one(
+        string="UoM",
+        comodel_name="uom.uom",
+    )
+    uom_min_quantity = fields.Float(
+        string="Min. Quantity in UoM",
+        digits="Product Unit of Measure",
+    )
+
+    @api.onchange(
+        "product_id",
+        "product_tmpl_id",
+    )
+    def onchange_product_uom(self):
```

Comment:
I would have used a compute/stored/readonly=False on `uom_id` field instead as onchanges tend to disappear.

Reply:
Right, done